xml/System.IO.Pipelines/PipeReader.xml

```diff
@@ -141,6 +145,8 @@ The <xref:System.IO.Pipelines.ReadResult.Buffer> previously returned from <xref:
 
 The examined data communicates to the pipeline when it should signal more data is available.
 
+The examined parameter should be greater than or equal to the examined position in the previous call to AdavanceTo. Otherwise <xref:System.InalidOperationException> will be thrown.
```

Comment:
```suggestion
The examined parameter should be greater than or equal to the examined position in the previous call to `AdvanceTo`. Otherwise, an <xref:System.InvalidOperationException> is thrown.
```